yarn.lock

```diff
@@ -4,23 +4,28 @@
 
 "@ministryofjustice/frontend@1.6.4":
   version "1.6.4"
-  resolved "https://registry.yarnpkg.com/@ministryofjustice/frontend/-/frontend-1.6.4.tgz#396ffdc1699253671c8bc93ffa879735b8d4e8da"
+  resolved "https://registry.npmjs.org/@ministryofjustice/frontend/-/frontend-1.6.4.tgz"
```

Comment:
This will annoy me, but I honestly have no idea why you are connected to a different MOJ registry... :(